library/core/src/result.rs

```diff
@@ -1113,11 +1113,15 @@ impl<T, E> Result<T, E> {
         }
     }
 
-    /// Returns the contained [`Ok`] value or a default
+    /// Consumes the `self` argument and returns the contained [`Ok`] value
+    /// or the default value.
     ///
-    /// Consumes the `self` argument then, if [`Ok`], returns the contained
-    /// value, otherwise if [`Err`], returns the default value for that
-    /// type.
+    /// If the result is [`Err`], the default value for the type that would
+    /// be contained on success is returned.
+    ///
+    /// Unlike [`unwrap`], this function will not panic.
```

Comment:
I think this should be placed in front of the "If the result is [`Err`]..." sentence, like "Unlike [`unwrap`], this function will not panic. Instead, if the result is...".

Reply:
I actually like this at the end.  It's not describing what the function *does*, it's just comparing it with other methods.  And "here's some related methods and why you might want them instead" is best *after* the direct description of what the method actually does.